docs/building/index.rst

```diff
@@ -102,6 +102,159 @@ Java
 
 If you're building OpenDDS for use by Java applications, please see the file :ghfile:`java/INSTALL` instead of this one.
 
+.. _building-sec:
+
+Security
+========
+
+..
+    Sect<14.1>
+
+:ref:`sec` is disabled by default, and must be enabled by passing ``--security`` to the configure script.
+
+It requires the following libraries to be available:
+
+- Xerces-C++ v3.x
+- OpenSSL v1.0.2+, v1.1, or v3.0.1+
```

Comment:
Should we consider moving these to some kind of file or is this the only file that contains these versions?

Future work.

Reply:
This information is already in the dependencies, so I will just link to Xerces and OpenSSL there. In the future though most of what I added here should probably be generalized and maybe moved to dependencies.

Reply:
Yeah, thinking about it more, unless it's trivial like package names or a special case, I don't really want to be in the business of telling people how to build and install FOSS that isn't ours.